src/quacc/atoms/skzcam.py

```diff
@@ -23,13 +26,751 @@
+                            basis=basis_set,
+                            frozencore=oniom_parameters["frozencore"],
+                            code=oniom_parameters["code"],
+                            ecp=oniom_parameters["ecp"],
+                            ri_scf_basis=oniom_parameters["ri_scf_basis"],
+                            ri_cwft_basis=oniom_parameters["ri_cwft_basis"],
+                        )
+
+                        # Use MRCCInputGenerator to generate the necessary blocks for the MRCC ASE calculator
+                        mrcc_block_inputs = MRCCInputGenerator(
+                            adsorbate_slab_embedded_cluster=self.adsorbate_slab_embedded_cluster,
+                            quantum_cluster_indices=self.quantum_cluster_indices_set[
+                                cluster_num - 1
+                            ],
+                            ecp_region_indices=self.ecp_region_indices_set[
+                                cluster_num - 1
+                            ],
+                            element_info=element_info,
+                            include_cp=True,
+                            multiplicities={
+                                "adsorbate_slab": 1,
+                                "adsorbate": 1,
+                                "slab": 1,
+                            },
+                        ).generate_input()
+
+                        if oniom_method == "deltaCC":
+                            mrcc_default_calc_inputs = {
+                                "calc": "LNO-CCSD(T)",
+                                "scftype": "rhf",
+                                "verbosity": 3,
+                                "mem": f"{oniom_parameters['max_memory']}MB",
+                                "symm": "off",
+                                "unit": "angs",
+                                "scfiguess": "small",
+                                "scfmaxit": 1000,
+                                "scfalg": "locfit1",
+                                "bpedo": 0.99999,
+                                "ccmaxit": 400,
+                                "usedisk": 0,
+                                "ccsdalg": "dfdirect",
+                                "ccsdthreads": 4,
+                                "ccsdmkl": "thr",
+                                "ptthreads": 4,
+                            }
+                        else:
+                            mrcc_default_calc_inputs = {
+                                "calc": "DF-MP2",
+                                "scftype": "rhf",
+                                "verbosity": 3,
+                                "mem": f"{oniom_parameters['max_memory']}MB",
+                                "symm": "off",
+                                "unit": "angs",
+                                "scfiguess": "small",
+                                "scfmaxit": 1000,
+                                "scfalg": "locfit1",
```

Comment:
All of this kind of stuff does not belong in `atoms/skzcam.py`. These are default parameters. They belong with the recipes. For instance, see https://github.com/Quantum-Accelerators/quacc/blob/main/src/quacc/recipes/qchem/core.py.